lobster/config/parser.py

```diff
@@ -102,13 +102,11 @@ def parse_level_declaration(self):
                        "duplicate declaration")
 
         item = {
-            "name"                   : level_name,
-            "kind"                   : level_kind,
-            "traces"                 : [],
```

Comment:
Note: if "dot" is not installed, then the bug will not be visible, because the tracing policy graph will not be rendered. See https://github.com/bmw-software-engineering/lobster/blob/f69c82ba408c7282fe48988b0a98ef106dee9816/lobster/tools/core/html_report.py#L365-L369